CMakeLists.txt

```diff
@@ -38,10 +39,23 @@ add_service_files(
   InitializeMap.srv
 )
 
+add_action_files(
+  FILES 
+  dock.action 
+  undock.action
+)
+
+generate_messages(
+  DEPENDENCIES 
+  actionlib_msgs
+  std_msgs
+  visualization_msgs
+)
+
 catkin_package(
   LIBRARIES image_sub_nodelet
```

Comment:
is nodelet required by the original docking code?

Reply:
I have the same question. Unless I am missing something, I don't see this code being loaded as a nodelet.

Reply:
yeah that's the original rover code.  should I change it?

Reply:
No don't bother in that case. Let's collect those cleanups for after the demo.